### PR TITLE
submissions: add literature sub endpoint

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: tests/integration/submissions/test_submissions_views.py [hunk 1]
```diff
@@ -155,3 +155,97 @@ def test_populate_and_serialize_data_for_submission(
...
+        headers=headers,
+    )
+    assert response.status_code == 200
+    mock_requests_post.assert_called_once()
+
+    post_args, post_kwargs = _get_args_and_kwargs_of_first_mock_call(mock_requests_post)
+    post_url = post_args[0]
+    post_data = json.loads(post_kwargs["data"])
+
+    assert post_url == f"{current_app.config['INSPIRE_NEXT_URL']}/workflows/literature"
```
Comment (drjova) [inline tests/integration/submissions/test_submissions_views.py]: I'd use directly the fixture app instead.
```suggestion
    assert post_url == f"{app.config['INSPIRE_NEXT_URL']}/workflows/literature"
```

File: tests/integration/submissions/test_submissions_views.py [hunk 2]
```diff
@@ -155,3 +155,97 @@ def test_populate_and_serialize_data_for_submission(
...
+                }
+            }
+        ),
+        headers=headers,
+    )
+    assert response.status_code == 503
+
+
+def _get_args_and_kwargs_of_first_mock_call(mock_function):
+    call = mock_function.call_args_list[0]
```
Comment (drjova) [inline tests/integration/submissions/test_submissions_views.py]:
```suggestion
args, kwargs  =mock_function.call_args_list[0]
return args, kwargs
```

File: inspirehep/submissions/views.py [hunk 3]
```diff
@@ -95,7 +95,37 @@ def _get_user_orcid():
...
+        data = {"data": serialized_data, "form_data": form_data}
+        response = requests.post(
+            f"{current_app.config['INSPIRE_NEXT_URL']}/workflows/literature",
+            data=json.dumps(data),
+            headers={"content-type": "application/json"},
+        )
+        if response.status_code == 200:
+            return response.content
+        else:
+            abort(503)
```
Comment (drjova) [inline inspirehep/submissions/views.py]:
```python
 if response.status_code == 200:
             return response.content
 abort(503)

```

File: tests/integration/submissions/test_submissions_views.py (hunk 1 above)
Comment (harunurhan) [reply tests/integration/submissions/test_submissions_views.py]: why?

**Pull request merged**